ipn/ipnlocal/local.go

```diff
@@ -665,6 +675,16 @@ func (b *LocalBackend) linkChange(delta *netmon.ChangeDelta) {
 	b.updateFilterLocked(b.netMap, b.pm.CurrentPrefs())
 	updateExitNodeUsageWarning(b.pm.CurrentPrefs(), delta.New, b.health)
 
+	// Upon any major link change, wait for 3 seconds for things to settle, then asynchronously run
```

Comment:
"major"? where are you checking that it's major?

this is for all link changes, no?

But ideally we'd only do this if our Tailscale control/derp connection(s) are dead and won't connect. Otherwise this seems like a lot of network noise all the time.